backend/app/routes/broadcast/getBroadcasts.js

```diff
@@ -0,0 +1,43 @@
+const to = require('await-to-js').default;
+const Broadcast = require('../../models/Broadcast');
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+
+module.exports = async (req, res, next) => {
+  const page = parseInt(req.query.page, 10);
+  const limit = parseInt(req.query.limit, 10);
```

Comment:
in the request you will never get the limit, you will only get page number

Reply:
how many results should I send?

Reply:
keep any reasonable number as a constant, maybe 10 or 5

Reply:
here 10 is passed to parseInt function so that the parsed value is a base 10 number and does not need to be changed.

Reply:
I have used a limit so the front end can get the no of results they want depending upon the situation. but yeah this can be omitted and can be kept constant for eg 10 as limit which will always send max of 10 results.